docs/source/concepts/dataset.md

```diff
@@ -28,6 +28,15 @@ Supports `[.png, .jpg, .jpeg]` file extensions.
                 - ...
             - ...
 
+#### Tabular Data
+Supports `.csv` file formats.
+
+Expects the folder path to point to a directory where the folder structure is the following:
+- Your folder
+  - file1.csv
+  - file2.csv
+
+You can have a single `.csv` file if you wish.
```

Comment:
no need to have this comment here, you should also have `...` below these files to show you can have multiple.

another small change: could you make `Your folder` be called `root` instead?